### DISCUSSION TITLE
Scale minio

### DISCUSSION THREAD

**Discussion by VA-PLE** [minio/minio#15236] [Q&A] [answered]
HI. I needed to increase the disk space on a current minio. now minio works on 3 dedicated servers (`MINIO_VOLUMES: "http://minio{1...3}myhost.io:9000/opt/disk{1...4}/minio"`).

I decided to add one more server with the configuration (`"http://minio{4...4}myhost.io:9000/opt/disk{1...4}/minio"`).

please tell me how to do it correctly

**Comment by klauspost** [comment, marked as answer]
I assume you've seen https://docs.min.io/minio/baremetal/installation/expand-minio-distributed.html

We recommend you expand with at least the amount of servers/drives you are running in your primary pool, or a multiple of it. 

In terms of availability your new server pool is independent, effectively being 1 server with 4 drives, which doesn't match the availability of your existing setup. Therefore "small" expansions are not a recommended approach.

 I recommend you expand with a setup like `http://minio{4...6}myhost.io:9000/opt/disk{1...4}/minio`, which will match your existing cluster.

**Comment by VA-PLE** [comment]
i just checked on a test environment, it seems fine. Previous attempts to add just one server resulted in the entire server pool crashing upon restart
